<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QVTKOpenGLNativeWidget" name="meshViewer" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout2" stretch="0,0">
        <item>
         <layout class="QVBoxLayout" name="meshesToolbar">
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearCanvasButton">
          <property name="text">
           <string>Clear canvas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenMesh"/>
    <addaction name="actionOpenAnnotations"/>
    <addaction name="actionSaveAnnotations"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVisibleSelection"/>
   <addaction name="actionRemoveSelected"/>
   <addaction name="separator"/>
   <addaction name="actionVerticesSelection"/>
   <addaction name="actionLinesSelection"/>
   <addaction name="actionTrianglesRectangleSelection"/>
   <addaction name="actionTrianglesLassoSelection"/>
   <addaction name="separator"/>
   <addaction name="actionSelectionAnnotation"/>
   <addaction name="actionselectAnnotations"/>
   <addaction name="actionEditAnnotations"/>
   <addaction name="actionAnnotationRelation"/>
  </widget>
  <action name="actionOpenMesh">
   <property name="text">
    <string>Open mesh</string>
   </property>
  </action>
  <action name="actionOpenAnnotations">
   <property name="text">
    <string>Open annotations</string>
   </property>
  </action>
  <action name="actionSaveAnnotations">
   <property name="text">
    <string>Save annotations</string>
   </property>
  </action>
  <action name="actionVerticesSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/point_rect.png</normaloff>:/Icons/point_rect.png</iconset>
   </property>
   <property name="text">
    <string>VerticesSelection</string>
   </property>
   <property name="toolTip">
    <string>Select vertices for annotation</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionTrianglesRectangleSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/triangle_rect.png</normaloff>:/Icons/triangle_rect.png</iconset>
   </property>
   <property name="text">
    <string>TrianglesRectangleSelection</string>
   </property>
   <property name="toolTip">
    <string>Select triangles with rectangle selector</string>
   </property>
  </action>
  <action name="actionTrianglesLassoSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/polygonalLasso.png</normaloff>:/Icons/polygonalLasso.png</iconset>
   </property>
   <property name="text">
    <string>TrianglesLassoSelection</string>
   </property>
   <property name="toolTip">
    <string>Select triangles with lasso selector</string>
   </property>
  </action>
  <action name="actionSelectionAnnotation">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/tag-icon.png</normaloff>:/Icons/tag-icon.png</iconset>
   </property>
   <property name="text">
    <string>SelectionAnnotation</string>
   </property>
   <property name="toolTip">
    <string>Annotate selected primitives</string>
   </property>
  </action>
  <action name="actionAnnotationRelation">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/relationship.png</normaloff>:/Icons/relationship.png</iconset>
   </property>
   <property name="text">
    <string>AnnotationRelation</string>
   </property>
   <property name="toolTip">
    <string>Create a relation between selected annotations (min 2 annotations)</string>
   </property>
  </action>
  <action name="actionRemoveSelected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/Eraser-icon.png</normaloff>:/Icons/Eraser-icon.png</iconset>
   </property>
   <property name="text">
    <string>RemoveSelected</string>
   </property>
   <property name="toolTip">
    <string>Remove primitives from selection</string>
   </property>
  </action>
  <action name="actionVisibleSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/visible_icon.png</normaloff>:/Icons/visible_icon.png</iconset>
   </property>
   <property name="text">
    <string>VisibleSelection</string>
   </property>
   <property name="toolTip">
    <string>Select only visible primitives</string>
   </property>
  </action>
  <action name="actionLinesSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/polyline.png</normaloff>:/Icons/polyline.png</iconset>
   </property>
   <property name="text">
    <string>LinesSelection</string>
   </property>
   <property name="toolTip">
    <string>Select edges for annotation</string>
   </property>
  </action>
  <action name="actionselectAnnotations">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/select_icon.png</normaloff>:/Icons/select_icon.png</iconset>
   </property>
   <property name="text">
    <string>selectAnnotations</string>
   </property>
   <property name="toolTip">
    <string>Select annotations on mesh</string>
   </property>
  </action>
  <action name="actionEditAnnotations">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/Icons/edit_icon.png</normaloff>:/Icons/edit_icon.png</iconset>
   </property>
   <property name="text">
    <string>EditAnnotations</string>
   </property>
   <property name="toolTip">
    <string>Edit selected annotation (only one per time)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
